docs/content/config/advanced/kubernetes.md

```diff
@@ -2,392 +2,375 @@
-                subPath: KeyTable
-                mountPath: /tmp/docker-mailserver/opendkim/KeyTable
-                readOnly: true
-              - name: config
-                subPath: TrustedHosts
-                mountPath: /tmp/docker-mailserver/opendkim/TrustedHosts
-                readOnly: true
-              - name: opendkim-keys
-                mountPath: /tmp/docker-mailserver/opendkim/keys
-                readOnly: true
-              - name: data
-                mountPath: /var/mail
-                subPath: data
-              - name: data
-                mountPath: /var/mail-state
-                subPath: state
-              - name: data
-                mountPath: /var/log/mail
-                subPath: log
-            ports:
-              - name: smtp
-                containerPort: 25
-                protocol: TCP
-              - name: smtp-secure
-                containerPort: 465
-                protocol: TCP
-              - name: smtp-auth
-                containerPort: 587
-              - name: imap
-                containerPort: 143
-                protocol: TCP
-              - name: imap-secure
-                containerPort: 993
-                protocol: TCP
-            envFrom:
-              - configMapRef:
-                  name: mailserver.env.config
-          volumes:
-            - name: config
-              configMap:
-                name: mailserver.config
-            - name: opendkim-keys
-              secret:
-                secretName: mailserver.opendkim.keys
-            - name: data
-              persistentVolumeClaim:
-                claimName: mail-storage
-            - name: tmp-files
-              emptyDir: {}
-    ```
+```yaml
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: mailserver.environment
+
+immutable: true # turn off during development
+
+data:
+  TLS_LEVEL: modern
+  POSTSCREEN_ACTION: drop
+  OVERRIDE_HOSTNAME: mail.example.com
+  FAIL2BAN_BLOCKTYPE: drop
+  POSTMASTER_ADDRESS: postmaster@example.com
+  UPDATE_CHECK_INTERVAL: 10d
+  POSTFIX_INET_PROTOCOLS: ipv4
+  ONE_DIR: '1'
+  DMS_DEBUG: '0'
+  ENABLE_CLAMAV: '1'
+  ENABLE_POSTGREY: '0'
+  ENABLE_FAIL2BAN: '1'
+  AMAVIS_LOGLEVEL: '-1'
+  SPOOF_PROTECTION: '1'
+  MOVE_SPAM_TO_JUNK: '1'
+  ENABLE_UPDATE_CHECK: '1'
+  ENABLE_SPAMASSASSIN: '1'
+  SUPERVISOR_LOGLEVEL: warn
+  SPAMASSASSIN_SPAM_TO_INBOX: '1'
+```
 
-!!! warning
-    Any sensitive data (keys, etc) should be deployed via [Secrets][k8s-config-secret]. Other configuration just fits well into [ConfigMaps][k8s-config-pod].
+We can also make use of user-provided configuration files, e.g. `user-patches.sh`, `postfix-accounts.cf` and more, to adjust `docker-mailserver` to our likings. We encourage you to have a look at [Kustomize][docs-kustomize] for creating `ConfigMap`s from multiple files, but for now, we will provide a simple, hand-written example. This example is absolutely minimal and only goes to show what can be done.
```

Comment:
```suggestion
We can also make use of user-provided configuration files, e.g. `user-patches.sh`, `postfix-accounts.cf` and more, to adjust `docker-mailserver` to our likings. We encourage you to have a look at [Kustomize][kustomize] for creating `ConfigMap`s from multiple files, but for now, we will provide a simple, hand-written example. This example is absolutely minimal and only goes to show what can be done.
```

Or if you prefer the `[Kustomize]` alone is fine with me too.

Reply:
I just now saw this, alright, I can commit your changes :D